CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.0.4-dev
```

Comment:
Do we consider supporting new language versions a "feature"?

```suggestion
# 2.1.0-dev
```

Reply:
In the past I haven't, but now that you mention it, that's a good idea.